src/css/base.scss

```diff
@@ -284,8 +288,8 @@ a:focus {
     top: 0;
 }
 
-.dc__left-35 {
```

Comment:
why is it removed?

Reply:
Maybe it was revolved while resolving conflicts in base.scss

Reply:
Maybe it was removed while resolving conflicts in base.scss